test/functional/wallet_fundrawtransaction.py

```diff
@@ -1522,5 +1524,20 @@ def test_input_confs_control(self):
 
         wallet.unloadwallet()
 
+    def test_cannot_cover_fees(self):
+        self.log.info("Test tx amount exceeds available balance when fees are included")
+
+        self.nodes[1].createwallet("cannot_cover_fees")
+        wallet = self.nodes[1].get_wallet_rpc("cannot_cover_fees")
+
+        self.nodes[0].sendtoaddress(wallet.getnewaddress(), 0.3)
+        self.generate(self.nodes[0], 1)
+
+        rawtx = wallet.createrawtransaction(inputs=[], outputs=[{self.nodes[0].getnewaddress(): 0.3}])
+        expected_err_msg = "The total transaction amount exceeds your balance when fees are included"
+        assert_raises_rpc_error(-4, expected_err_msg, wallet.fundrawtransaction, rawtx)
```

Comment:
Could this test case be extended by attempting the same transaction with SFFO and showing that it succeeds?